CHANGELOG.md

```diff
@@ -62,6 +62,8 @@ when necessary—we also advise to not ignore `DeprecationWarning`s.
   evaluation; instead, its dataset field now holds an xarray dataset (instead 
   of a path), which does not change over the instance lifetime. 
   This reduces the amount of time spent on I/O ({ghpr}`212`).
+* Re-formated `spectra/particles/govaerts_2021-*-extrapolated.nc` data sets
+  ({ghpr}`213`).
```

Comment:
Please move this entry at the bottom of the section (this maintains an approximate chronological order).